code-challenges.test.js

```diff
@@ -34,9 +60,30 @@ const hodgepodge1 = [23, "Heyyyy!", 45, -10, 0, "Yo", false]
 const hodgepodge2 = [5, "Hola", 43, -34, "greetings", true]
 // Expected output: [ 2, 1, -1 ]
 
+describe ("remainder" = () => {
+  it ("takes in a mixed data array and returns an array of only the REMAINDERS of the numbers when divided by 3", ()=>{
+   expect(remainder(hodgepodge1)).toEqual([ 2, 0, -1, 0 ]);
+   expect(remainder(hodgepodge2)).toEqual([ 2, 1, -1 ]);
+  })
+})
+
 
 // b) Create the function that makes the test pass.
 
+// create a new array
+// filter through the given array and determine which ones are numbers
+// if the current value is a number, get value % 3 and push it into the new array
+
+const remainder = (arr) => {
+  let numArr = [];
+  arr.filter((value) => {
+    if (typeof value === "number") {
+      numArr.push(value % 3);
+    }
+  });
+  return numArr;
+};
```

Comment:
Nice job! The cool thing about `filter` is you don't need a new array or `push` as it will always return an array.